READMEs/README_zh-TW.md

```diff
@@ -0,0 +1,62 @@
+# ReZygisk
+> 繁體中文（README_zh-TW.md）是根據[英文版自述檔案（README.md）](https://github.com/PerformanC/ReZygisk/blob/main/README.md)翻譯，僅供參考以便理解英文內容，翻譯可能滯後。
+
+ReZygisk 是 Zygisk Next 的一個分支，這是一個獨立實現的 Zygisk，為 KernelSU、Magisk（除了內建支援外）和 APatch（開發中）提供 Zygisk API 支援。
+
+此專案致力於用 C 語言重寫原有的 C++ 和 Rust 代碼，藉此以更現代且高效的方式實現 Zygisk API，並採用更寬鬆的授權條款。
+
+> [!NOTE]
+> 此模組/分支仍在開發中（WIP）；請僅安裝正式版本的壓縮包。
+>
+> 雖然你可以從 [Actions](https://github.com/PerformanC/ReZygisk/actions) 頁面安裝 .zip 檔，但若因此導致裝置進入開機循環（Bootloop），後果須自行承擔。
+
+## 為什麼選擇ReZygisk？
+
+最新版本的 Zygisk Next 已轉為閉源，只有核心開發者能查閱完整的原始碼。這不僅限制了其他開發者的貢獻，也無法進行代碼審計。由於 Zygisk Next 是一個以超級使用者（root）權限運行的模組，能夠存取整個系統，若閉源將帶來重大的安全風險。
+
+雖然 Zygisk Next 的開發者在 Android 社群中享有盛譽，並且備受信任，但這並不代表他們的專案就完全沒有任何惡意程式碼或漏洞。我們（PerformanC）理解他們因某些原因選擇保持閉源，但我們堅信開源才是更好的選擇。
+
+## 優勢
+
+- 永遠是自由及開放原始碼軟體（FOSS）
+
+## 依賴項
+
+| 工具            | 說明                                   |
+|-----------------|---------------------------------------|
+| `Android NDK`   | Android 原生開發工具包                  |
+
+### C++ 依賴項
+
+| 依賴     | 說明                                          |
+|----------|----------------------------------------------|
+| `lsplt`  | Android 的簡單 PLT（程式連結表） 勾取            |
```

Comment:
```suggestion
| `lsplt`  | Android 的簡單 PLT（程式連結表） 勾取           |
```

Oh, I mean this.

Reply:
Please revert the last commit if possible: git reset --soft HEAD~1 and unstage the changes (the command to do so shows in git status)